core/settings.py

```diff
@@ -92,7 +92,7 @@
 # https://docs.djangoproject.com/en/2.1/ref/settings/#databases
 
 DATABASES = {
-    'default': env.db('DATABASE_URL', default='postgres://cmput404:abramiscool123!@cmput404-dev.c1dsguk3kuvt.us-west-2.rds.amazonaws.com:5432/404posts'),
+    'default': env.db('DATABASE_URL', default='postgres://cmput404:abramiscool123!@cmput404-dev.c1dsguk3kuvt.us-west-2.rds.amazonaws.com:5432/testing'),
```

Comment:
When running tests this isn't necessary, pytest should automatically switch DBs, unless I'm misunderstanding what this is for

Reply:
This is where posts work, posts don't work on the other one